docs/developer/reference/services/fetch-service.rst

```diff
@@ -53,22 +53,25 @@ The life-cycle of a fetch-service session within Launchpad goes as follows:
 
 Using the fetch service
 ~~~~~~~~~~~~~~~~~~~~~~~
-Currently, the fetch service can only be used for building snaps, rocks and
-source packages.
+Currently, the fetch service can only be used for building snaps, charms, rocks 
+and source packages.
 
-To do so, a Launchpad admin is required to set the
+For `Snaps`, this option is now available in `Edit recipe` instead of
+`Administer recipe`. For other types, a Launchpad admin is required to set the
```

Comment:
I would remove the "instead of" and the reference to `Administer recipe`, it's part of the admin area so normal users shouldn't need to know this.